client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -14,24 +14,27 @@ import config from 'config';
 import Button from 'components/button';
 import { cancelAndRefundPurchase, cancelPurchase, submitSurvey } from 'lib/upgrades/actions';
 import { clearPurchases } from 'state/purchases/actions';
-import {
-	isHappychatAvailable,
-	isHappychatChatActive,
-} from 'state/happychat/selectors';
+import { isHappychatAvailable, isHappychatChatActive } from 'state/happychat/selectors';
 import { connect } from 'react-redux';
 import Dialog from 'components/dialog';
 import CancelPurchaseForm from 'components/marketing-survey/cancel-purchase-form';
-import enrichedSurveyData from 'components/marketing-survey/cancel-purchase-form/enrichedSurveyData';
-import initialSurveyState from 'components/marketing-survey/cancel-purchase-form/initialSurveyState';
+import enrichedSurveyData
+	from 'components/marketing-survey/cancel-purchase-form/enrichedSurveyData';
+import initialSurveyState
+	from 'components/marketing-survey/cancel-purchase-form/initialSurveyState';
```

Comment:
This is because prettier ensures that line is not longer than 100 chars or sth like that. I don't remember what @samouri set exactly. Question here is rather why we have so long file path? 😄